exec/cnex/cnex.c

```diff
@@ -125,15 +167,15 @@ int main(int argc, char* argv[])
     long nSize = ftell(fp);
     fseek(fp, 0, SEEK_SET);
 
-    TBytecode *pModule = bytecode_newBytecode();
-    uint8_t *bytecode = malloc(nSize);
-    if (bytecode == NULL) {
+    TModule *pModule = module_newModule("");
+    pModule->code = malloc(nSize);
```

Comment:
Why is this done outside the `module_newModule` function? This breaks encapsulation.

Reply:
I'm not certain that it IS necessary to keep the raw bytes around.  I think that was just part of the design; just clean up everything during shutdown, rather than before.